README.md

```diff
@@ -28,6 +28,7 @@ Following environment variables are used by the software.
 * **CLIENT_SECRET** Client secret for your application.
 
 **Optional**
+* **HOSTNAME** Hostname to listen for requests. Defaults to all interfaces (0.0.0.0).
```

Comment:
Minor, but  I'd reword it like so:

> Defaults to all IPv4/6 interfaces (0.0.0.0, ::).

so that others know that the server will bind by default on their IPv6 interfaces

Reply:
Noted!
Will change.